packages/babel-jest/src/__tests__/getCacheKey.test.ts

```diff
@@ -9,7 +9,7 @@ import type {TransformOptions as BabelTransformOptions} from '@babel/core';
 import type {TransformOptions} from '@jest/transform';
 import babelJest from '../index';
 
-const {getCacheKey} = babelJest.createTransformer();
+const {getCacheKey} = await babelJest.createTransformer();
```

Comment:
We don't have TLA

Reply:
@SimenB Right, I forgot to typecheck tests. There was also a failure with windows but I'm not sure what it was. Thanks for your help on this!

Reply:
Windows is a bit flaky